corehq/apps/fixtures/upload/run_upload.py

```diff
@@ -191,9 +192,20 @@ def _update_progress(table_count, item_count, items_in_table):
                                                    transaction=transaction)
 
     clear_fixture_quickcache(data_types)
+    clear_fixture_blob(domain, data_types)
     return return_val
 
 
+def clear_fixture_blob(domain, data_types):
+    """Clear global fixture cache for the domain f any of the data_types uploaded
+    are global
+
+    """
+    if any([data_type.is_global for data_type in data_types]):
```

Comment:
nitpick: rather than creating an intermediate list with a list comprehension, `any()` is marginally more efficient with a generator expression, like this:

```py
any(data_type.is_global for data_type in data_types)
```